WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListFragment.java

```diff
@@ -103,10 +103,10 @@ public void onClick(View view) {
     public void onAttach(Context context) {
         super.onAttach(context);
 
-        if (context instanceof PublicizeManageConnectionsListener) {
-            mListener = (PublicizeManageConnectionsListener) context;
+        if (context instanceof PublicizeButtonPrefsListener) {
+            mListener = (PublicizeButtonPrefsListener) context;
```

Comment:
This listener will never get set on API < 23, [relevant SO question](https://stackoverflow.com/questions/32604552/onattach-not-called-in-fragment). You'll have to use `onAttach(Activity activity)` to support earlier API's.